src/qt/sendcoinsdialog.cpp

```diff
@@ -318,11 +318,24 @@ void SendCoinsDialog::on_sendButton_clicked()
         // append fee string if a fee is required
         questionString.append("<hr /><span style='color:#aa0000;'>");
         questionString.append(BitcoinUnits::formatHtmlWithUnit(model->getOptionsModel()->getDisplayUnit(), txFee));
+
         questionString.append("</span> ");
         questionString.append(tr("added as transaction fee"));
 
+        double txSize = (double)currentTransaction.getTransactionSize();
+
+        // safe to assume that txSize cannot be 0 at this point?
+        double satPerVByte = txFee / txSize; // txFee is already in satoshis
+
         // append transaction size
-        questionString.append(" (" + QString::number((double)currentTransaction.getTransactionSize() / 1000) + " kB)");
+        questionString.append(" (");
+        questionString.append(GUIUtil::formatBytes(txSize));
```

Comment:
This may loos precision since formatBytes() does show 1.2kb as 1kb... not sure if this is a problem though. Maybe better keep it the way it was.

Reply:
Thanks for the heads up, moved back to original version in e8485d10e63c1c66ea85b253ff38ee86fd2a3ea6